src/components/Forms/EpicNominationForm/NominateRatForm.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable react/jsx-no-bind */
```

Comment:
This isn't an eslint rule to be ignored. 
Please consult me for a solution before ignoring rules like this.

Callbacks defined in function scope which are used in JSX or passed to other hooks should be wrapped with the `useCallback()` hook so they are provided with a stable function instance.